### DISCUSSION TITLE
today the plugin does not start

### DISCUSSION THREAD

**Discussion by santta** [rsjaffe/MIDI2LR#660] [Problems]
**Additional context**
Good afternoon, I have been using your application for a long time, but today the plugin does not start, it is displayed as enable in the plugin manager, but does not start itself, and  does not start after pressing the application launch button from the lightroom menu, and from the exe file. I tried to figure out what the problem was, and updated the lightroom to version (11.3.1) and the application to (5.3.0.2). But the application also does not start.    The button for saving the log diagnostic file is not active.

**System(please complete the following information):**
 - OS: Windows 10 x64
 - Lightroom version 11.3.1
 - MIDI2LR version 5.3.0.2

**Screenshots**
![Безымdddянный](https://user-images.githubusercontent.com/45362973/172164443-995e1b91-4cf9-480e-9159-c991e0afbbac.png)

[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/8852148/MIDI2LR.log)

**Comment by rsjaffe** [comment]
That button is disabled because the plugin is functioning. Include the contents of the log file (MIDI2LR.log) in your message. This file is found by using the Lightroom menus: Help → Plug-in Extras MIDI2LR → Generate diagnostic report.

**Reply by santta** [reply]
[MIDI2LR.log](https://github.com/rsjaffe/MIDI2LR/files/8852142/MIDI2LR.log)

**Comment by rsjaffe** [comment]
Delete C:\Users\Admin\AppData\Roaming\MIDI2LR\
Delete MIDI2LR.lrplugin
Reinstall MIDI2LR

**Comment by santta** [comment]
it helped thanks
